### DISCUSSION TITLE
Menubar direction : top ?

### DISCUSSION THREAD

**Discussion by KaKi87** [jarek-foksa/xel#118] [Q&A] [answered]
I would like to put a hamburger menu in the bottom-right corner of my app, which means I need its direction to be *top*.

How to do that ?

Thanks

**Comment by jarek-foksa** [comment]
The menu should be automatically positioned above the hamburger button if there is not enough space below it (e.g. because the hamburger button is placed in the bottom-right corner of the app window).

**Reply by KaKi87** [reply]
Unfortunately, that's not happening, I'll attach a screenshot in a few minutes.

**Reply by KaKi87** [reply]
![eq4ze3f58ze7f](https://user-images.githubusercontent.com/21284089/183264510-f44c5ea4-6208-4032-b6d2-3c11fa3e2da5.png)

**Comment by jarek-foksa** [comment]
How are you positioning the button? It works as expected for me:

![2022-08-06 23 10 54](https://user-images.githubusercontent.com/193371/183267552-e19f2918-03c4-4c57-8fe8-504a0ee0b4e2.gif)

**Reply by KaKi87** [reply]
I'm using `flex`.

**Reply by KaKi87** [reply]
[Here](https://jsitor.com/ghZq5Bz2D)'s a minimal reproducible example. Thanks

**Comment by jarek-foksa** [comment, marked as answer]
You should use `<x-button>` to implement the hamburger menu, not `<x-menubar>`. Please check the updated example: https://jsitor.com/ghZq5Bz2D

**Reply by KaKi87** [reply]
That works, thanks. Too bad `x-button::part(arrow) { display: none; }` is needed though.
